### PR TITLE
add fontconfig and bzip2 as direct dependencies for Qt5 to fix installation with --rpath

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

This fixes the failing RPATH sanity check when `Qt5` is installed with `eb --rpath`:

```
One or more required libraries not found for /software/Qt5/5.14.1-GCCcore-9.3.0/lib/libQt5WebEngineCore.so.5.14.1
...
libfontconfig.so.1 => not found
```

When `fontconfig` is a direct dependency, a corresponding `-L` flag is included in `$LDFLAGS`, which is picked up by the RPATH wrapper to inject the equivalent `-Wl,-rpath=...` option.

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS=--rpath --installpath /tmp/pr11183

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3404.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/d83076d539c5d0db7f94c06b82122d8f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2457.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/4219bdf3a4d71dc93d3bc7bfb436453c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3149.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/299b6812a1da46f3a0e11e6928d96a4a for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS=--rpath --installpath /tmp/pr11183

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2457.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/ff74d32cd73ad17a2a1818fe54a79025 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3149.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ea3dafeba9ef7adb902929a97c434b89 for a full test report.

File: easybuild/easyconfigs/q/Qt5/Qt5-5.12.3-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -50,6 +50,7 @@ dependencies = [
     ('libjpeg-turbo', '2.0.2'),
     ('NSS', '3.42.1'),
     ('snappy', '1.1.7'),
+    ('bzip2', '1.0.6'),
```
Comment (ocaisa) [inline easybuild/easyconfigs/q/Qt5/Qt5-5.12.3-GCCcore-8.2.0.eb:53]: Missing `fontconfig/2.13.1` dep here

**Review by ocaisa — CHANGES_REQUESTED** [review]

**Review by ocaisa — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2700.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/77e3ab051619dfdb5d53311555677ff4 for a full test report.

**Pull request merged**